packages/create-ima-app/examples/hello/config/services.js

```diff
@@ -11,8 +11,10 @@ export default (ns, oc, config) => {
   $window.bindEventListener($window.getWindow(), 'error', event => {
     let error = event.error;
 
-    $router.handleError({ error }).catch(fatalError => {
-      config.$IMA.fatalErrorHandler(fatalError);
-    });
+    if (typeof $router?.handleError?.catch === 'function') {
```

Comment:
only `$router?.handleError`

Reply:
I don't think that will work since the `typeof Promise === 'object'`... or do you mean to change it to some more sophisticated promise check?

Reply:
$router instance has handleError function which return Promise https://github.com/seznam/ima/blob/v17.12.3/packages/core/src/router/AbstractRouter.js#L313 but you don't call function. Why do you add  condition?

Reply:
Fixed in https://github.com/seznam/ima/pull/175/commits/bf8f7f628b4a77abff8a89306fbd1adf6bfb863a